frontend/packages/knative-plugin/src/components/add/event-sources/CamelSection.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import YAMLEditorField from '@console/shared/src/components/formik-fields/YAMLEditorField';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+
+const CamelSection: React.FC = () => {
```

Comment:
Why is it specifically named `CamelSection`? I think this should be a generic `YAMLSection` where you take `title` as a prop based on the event source type. This should also work for dynamic event sources.

Reply:
created YAMLEditorSection to take title from formik values